onnxruntime/test/platform/ios/ios_package_test/OnnxRuntimeBase.podspec.template

```diff
@@ -0,0 +1,16 @@
+Pod::Spec.new do |spec|
+  spec.name         = "OnnxRuntimeBase"
+  spec.version      = "${ORT_VERSION}"
+  spec.summary      = "Onnx Runtime C/C++ Package"
+  spec.description  = <<-DESC
+  Onnx Runtime C/C++ framework pod.
+                   DESC
+
+  spec.homepage     = "https://github.com/microsoft/onnxruntime"
+  spec.license      = { :type => 'MIT' }
+  spec.authors      = { "ONNX Runtime" => "onnxruntime@microsoft.com" }
+  spec.platform     = :ios
+  spec.platform     = :ios, '13.0'
+  spec.source       = { :http => 'file:' + '${ORT_BASE_FRAMEWORK_ARCHIVE}' }
```

Comment:
if we include 'file:' in the templated value, we could replace it with the uploaded package URL later. assuming we want to use the same podspec

Reply:
I think we will use a different podspec for the actual package, this .podspec file is in the test project